app/src/main/res/layout/fragment_stock_exchange.xml

```diff
@@ -236,4 +237,39 @@
         android:orientation="vertical"
         app:layout_constraintGuide_percent="0.6" />
 
+    <LinearLayout
```

Comment:
I think we can do without LinearLayout

Reply:
yes i have removed linear layout and use the regular style followed in the app